plugins/services/src/js/components/SDKPlansTab.tsx

```diff
@@ -77,12 +77,19 @@ class SDKPlansTab extends React.PureComponent<SDKPlansTabProps, {}> {
       return null;
     }
     const { service, schedulerTaskId } = this.props;
+    let serviceIdURI, schedulerTaskIdURI;
+
+    try {
```

Comment:
there already is `if(!this.props.schedulerTaskID)` clause above, what about moving if after the deconstruction and just extending that one with ` || !service || !service.id`? 🤔 
```suggestion
const { service, schedulerTaskId } = this.props;

if(!service || !service.id || !schedulerTaskId) {
  return null;
}
```

Reply:
I think there is an issue with not just falsy values getting encoded, but invalid ones as well. encodeURIComponent will raise a URIError if it can't be encoded for some reason

Reply:
I agree with @brandonc . I don't know exactly why this occurs, but the problem is not that these are null or undefined (`encodeURIComponent(null)` and `encodeURIComponent(undefined)` will give no error, but rather `"null"` and `"undefined"`), but that they are invalid URI - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Malformed_URI#Encoding .